README.md

```diff
@@ -12,7 +12,7 @@ SOES is an EtherCAT slave stack written in c. Its purpose is to learn and
 to use. All users are invited to study the source to get an understanding
 how an EtherCAT slave functions.
 
-Features as of 2.1.2:
```

Comment:
Could we reword this to avoid the version number? Eventually someone will forget to update it here also.